src/css/imports/mixins.less

```diff
@@ -317,78 +314,75 @@
-        bottom: auto;
-      }
-
-      &:not(.jw-flag-time-slider-above) {
-        .jw-controlbar {
-          height: @controlbar-height;
-
-          .jw-overlay {
-            bottom: @controlbar-height;
-          }
+        .jw-controlbar .jw-controlbar-center-group .jw-text-alt  {
+            top: 50%;
+            transform: translateY(-50%);
+            margin: auto;
+            bottom: auto;
         }
 
-        .jw-icon-inline,
-        .jw-icon-tooltip,
-        .jw-text-elapsed,
-        .jw-text-duration,
-        .jw-text-countdown {
-          height: @controlbar-height;
-          line-height: @controlbar-height;
-        }
-      }
+        &:not(.jw-flag-time-slider-above) {
+            .jw-controlbar {
+                height: @controlbar-height;
 
-      &:not(.jw-flag-time-slider-above):not(.jw-flag-small-player) {
+                .jw-overlay {
+                    bottom: @controlbar-height;
+                }
+            }
 
-        .jw-time-tip {
-            bottom: (@controlbar-height / 2);
+            .jw-icon-inline,
+            .jw-icon-tooltip,
+            .jw-text-elapsed,
+            .jw-text-duration,
+            .jw-text-countdown {
+                height: @controlbar-height;
+                line-height: @controlbar-height;
+            }
         }
 
-        .jw-captions {
-            max-height: calc(97% - 6.25 * (@controlbar-height));
-        }
+        &:not(.jw-flag-time-slider-above):not(.jw-flag-small-player) {
+            .jw-time-tip {
+                bottom: (@controlbar-height / 2);
+            }
 
-        .jwplayer {
-            video::-webkit-media-text-track-container {
+            .jw-captions {
                 max-height: calc(97% - 6.25 * (@controlbar-height));
             }
-        }
 
-        .jw-controls {
-            .jw-controls-right {
-                bottom: @controlbar-height;
+            .jwplayer {
+                video::-webkit-media-text-track-container {
+                    max-height: calc(97% - 6.25 * (@controlbar-height));
+                }
             }
-        }
 
-        &.jw-flag-audio-player {
+            .jw-controls {
+                .jw-controls-right {
+                    bottom: @controlbar-height;
+                }
+            }
 
-          .jw-controlbar {
-            height: 100%;
-          }
+            &.jw-flag-audio-player {
+                .jw-controlbar {
+                    height: 100%;
+                }
 
-          .jw-icon-inline,
-          .jw-icon-tooltip,
-          .jw-text-elapsed,
-          .jw-text-duration,
-          .jw-text-countdown {
-              height: auto;
-              line-height: normal;
-          }
-        }
+                .jw-icon-inline,
+                .jw-icon-tooltip,
+                .jw-text-elapsed,
+                .jw-text-duration,
+                .jw-text-countdown {
+                    height: auto;
+                    line-height: normal;
```

Comment:
@jw-kaurand this causes the rewind button to be cropped in iOS Safari. It needs to be an explicit line-height.